src/config-import.js

```diff
@@ -193,6 +187,26 @@ export async function readConfig(configPath) {
             presetValue[key] = preset[key]
           }
         }
+        // check that preset references existing icon
+        const iconFilenames = new Set(
+          iconEntries.map((icon) => {
+            const matches = path
+              .basename(icon.filename)
+              .match(/([a-zA-Z--0-]+)-/)
```

Comment:
This regex has several problems. For example, it doesn't match `ok1-x.png`, and matches `ok./-x.png`. Could we repurpose the filename parsing from `parseIcon` instead?